src/org/labkey/test/tests/AbstractExportTest.java

```diff
@@ -51,17 +51,35 @@ public abstract class AbstractExportTest extends BaseWebDriverTest
     protected DataRegionTable dataRegion;
     protected DataRegionExportHelper exportHelper;
 
-    /** Return true if the rows can be selected in the grid. */
+    /**
+     * Return true if the rows can be selected in the grid.
+     */
     protected abstract boolean hasSelectors();
+
+    protected abstract boolean hasBrokenLookup();
+
     protected abstract String getTestColumnTitle();
+
+    protected abstract String getTestLookUpColumnHeader();
+
+    protected abstract int getTestLookUpColumnIndex();
```

Comment:
Consider making these non-abstract so that not all subclasses need to implement them.
```suggestion
    protected String getTestLookUpColumnHeader()
    {
        return null;
    }

    protected int getTestLookUpColumnIndex()
    {
        return -1;
    }
```

Reply:
plan was to edit the rest of the sample files for other grids to test the same broken lookup case.